pub_sub/csharp/checkout/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Dapr.Client;
+using System.Threading;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+
+while(true) {
+    Random random = new Random();
+    var order = new Order(random.Next(1,1000));
+    //var data = JsonSerializer.Serialize<Order>(order);
+    CancellationTokenSource source = new CancellationTokenSource();
+    CancellationToken cancellationToken = source.Token;
+    using var client = new DaprClientBuilder().Build();
+    await client.PublishEventAsync("order_pub_sub", "orders", order, cancellationToken);
+    //await client.PublishEventAsync("order_pub_sub", "orders", data, cancellationToken);
+    Console.WriteLine("Published data: " + order);

```

Comment:
This is going to print the class name `Order`. Is that what you wanted?

Reply:
It is printing name and the content of the anonymous type which seems to work for this. What would work better? 

== APP == Published data: Order { order_id = 694 }